product_docs/docs/pgd/5.6/ddl/ddl-command-handling.mdx

```diff
@@ -253,11 +253,11 @@ Some variants of `ALTER TABLE` currently aren't allowed on a PGD node:
 -   `ADD COLUMN ... DEFAULT (non-immutable expression)` &mdash; This is not allowed because
     it currently results in different data on different nodes. See
     [Adding a column](ddl-workarounds/#adding-a-column) for a suggested workaround.
-    This can be overriden using `bdr.permit_unsafe_commands` if user is sure the command is
+    This can be overridden using `bdr.permit_unsafe_commands` if user is sure the command is
```

Comment:
Is there some reason this is in third person and passive voice? Suggest this revision:

```suggestion
    You can override this behavior using `bdr.permit_unsafe_commands` if you're sure the command is
```